README.md

```diff
@@ -1,36 +1,90 @@
 [![Generic badge](https://img.shields.io/github/workflow/status/mrtolkien/leaguepedia_parser/Python%20application)](https://shields.io/)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-# leaguepedia_parser
-A parser for Leaguepedia focused on accessing esports data.
+# Leaguepedia Parser
 
-It is very minimal at the moment and focused on my own usage of Leaguepedia’s data. Pull requests to add features are
-more than welcome! 
+A parser for Leaguepedia focused on accessing esports data, which largely keeps game data in the [community-defined LoL DTO format](https://github.com/mrtolkien/lol_dto).
+
+It's very minimal at the moment and focused on my own usage of Leaguepedia’s data.
+Pull requests to add features are more than welcome! (See ["Contributing"](https://github.com/mrtolkien/leaguepedia_parser#Contributing))
 
 # Install
+
 `pip install leaguepedia_parser`
 
 # Demo
+
 ![Demo](leaguepedia_parser_demo.gif)
 
 # Usage
-```python
-import leaguepedia_parser
 
-# Gets regions names
-regions = leaguepedia_parser.get_regions() 
+Use the following code-blocks as examples if you want to get:
+
+- leaguepedia_parser imported
+
+  ```python
+  import leaguepedia_parser as lpp
```

Comment:
Importing modules as aliases is a poor coding practice. Even though it makes it slightly easier for you, it really hurts code readability for newcomers.

Here is the relevant google python style guide paragraph about imports: https://google.github.io/styleguide/pyguide.html#224-decision

I advise reading the full style guide, it’s not very long and a good read!

Reply:
Ah, I didn't know that. I've seen it done for long named modules in a lot of the python tutorials I've read, so I thought it was a normal practice.

Reading it as we speak-- I always prefer to learn the best long-term practices, so please do continue to point me to info/references or anything that's not ideal.

Reply:
Not **everybody** agrees with the google python style guide, but I feel like it’s one of the most comprehensive ones out there and it’s generally properly explained, which is why I favor it.

Reply:
Regarding shorthand for module names, as the google style guide says it’s ok for the most popular modules where the shorthand is generally agreed upon.

So I wouldn’t say anything for stuff `pandas as pd`, `numpy as np`, `matplotlib.image as mpimg`, `matplotlib.plot as plt` and so on and so forth. But this module isn’t popular enough for a shorthand yet!

Reply:
Yeah, that's what I figured from it as well. And, upon reflection, it's not even that much help to short-hand it since I'm sure most people use autocomplete in their code editors anyways.